public/v4/apps/squidex.yml

```diff
@@ -0,0 +1,114 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        depends_on:
+            - $$cap_appname-mongo
+        image: 'squidex/squidex:$$cap_squidex_version'
+        volumes:
+            - '$$cap_appname-data:/app/Assets'
+        restart: always
+        environment:
+            URLS__BASEURL: 'https://$$cap_url'
+            URLS__ENFORCEHTTPS: 'True'
+            EVENTSTORE__MONGODB__CONFIGURATION: 'mongodb://root:$$cap_mongo_password@srv-captain--$$cap_appname-mongo'
+            STORE__MONGODB__CONFIGURATION: 'mongodb://root:$$cap_mongo_password@srv-captain--$$cap_appname-mongo'
+            IDENTITY__ADMINEMAIL: $$cap_admin_email
+            IDENTITY__ADMINPASSWORD: $$cap_admin_password
+            IDENTITY__GOOGLECLIENT: $$cap_google_client
+            IDENTITY__GOOGLESECRET: $$cap_google_secret
+            IDENTITY__GITHUBCLIENT: $$cap_github_client
+            IDENTITY__GITHUBSECRET: $$cap_github_secret
+            IDENTITY__MICROSOFTCLIENT: $$cap_microsoft_client
+            IDENTITY__MICROSOFTSECRET: $$cap_microsoft_secret
+            LETSENCRYPT_HOST: $$cap_url
+            LETSENCRYPT_EMAIL: $$cap_admin_email
+            VIRTUAL_HOST: $$cap_url
+        caproverExtra:
+            containerHttpPort: '80'
```

Comment:
80 is default. You don't need to specify it.

Reply:
Removed